src/stream/src/executor/global_simple_agg.rs

```diff
@@ -271,15 +255,16 @@ impl<S: StateStore> GlobalSimpleAggExecutor<S> {
             info,
             input_pk_indices,
             input_schema,
-            mut agg_state,
             agg_calls,
             extreme_cache_size,
             mut agg_state_tables,
             mut result_table,
             mut state_changed,
         } = self;
-        let mut input = input.execute();
 
+        let mut agg_group = None;
```

Comment:
If we explicitly initialize it here before calling `apply_chunk`, maybe there's no need to pass an `Option` and `unwrap_mut` everywhere, which looks nicer. Anyway, it's not a big deal.

Reply:
Current behavior is like delaying the creation of `AggGroup` to the first `apply_chunk`. If no `apply_chunk` ever called during one epoch, `AggGroup` doesn't need to be created.